docs/schema/gcp.md

```diff
@@ -487,6 +489,43 @@ Representation of a GCP [Firewall](https://cloud.google.com/compute/docs/referen
     ```
 
 
+## GCPForwardingRule
+
+Representation of GCP [Forwarding Rules](https://cloud.google.com/compute/docs/reference/rest/v1/forwardingRules/list) and [Global Forwarding Rules](https://cloud.google.com/compute/docs/reference/rest/v1/globalForwardingRules/list).
```

Comment:
[question] I don't know GCP well enough, but would there be a benefit in separating global forwarding rules from regional forwarding rules? If so, should we represent them as separate node types, or should we have a field on the node to indicate that they are different?